install.sh

```diff
@@ -107,9 +107,9 @@ fetch() {
   curl --proto '=https' --tlsv1.2 -SfL --progress-bar -o "${dest}" "${url}"
 }
 
-ensure_cmd $([ "${OS}" == "macos" ] && echo "shasum" || echo "sha256sum")
+ensure_cmd $([ "${OS}" = "macos" ] && echo "shasum" || echo "sha256sum")
```

Comment:
Beyond the colored logging not working - fixes above - there were a few warnings / errors from `[[` and `==` bashisms missed in the 1st round.